package.json

```diff
@@ -18,41 +18,41 @@
     "node": ">=6"
   },
   "dependencies": {
-    "async": "^2.0.1",
-    "body-parser": "^1.12.4",
+    "async": "^2.6.1",
```

Comment:
When updating dependencies, we usually update only semver-major numbers and leave semver-minor/patch numbers untouched - the carrot operator `^` will take care of them. 

Please note that `npm update` does not update across semver-major numbers. If this commit is the outcome of `npm update`, then you need to check if there aren't any dependencies that need manual update.

Instructions:

1. Run `npm outdated --depth 0` to find which dependencies are outdated.
2. Run `npm install {dep}@latest` to update the dependency called `{dep}`.